### DISCUSSION TITLE
Is there a way to make the white mode the default?

### DISCUSSION THREAD

**Discussion by halilakin** [roam-garden/i#66] [Q&A]
I don't really like how black mode looks (the dots are invisible) and I don't like that it comes as default. Is there a way to make the white mode default?

**Comment by Stvad** [comment]
The mode is automatically selected based on the system state (e.g. if your system is in dark mode - it'd be dark by default). After you choose the mode explicitly - it'd be remembered

**Reply by Stvad** [reply]
Also I like dark mode colors here https://jayfarei.roam.garden/

**Reply by halilakin** [reply]
Me too! Can we make this default :)
